site/public/css/rainbow-customization.css

```
@@ -96,6 +97,32 @@
     display: none;
     margin-top: 10px;
     margin-bottom: 10px;
+    overflow: auto;
+}
+
+/* status-log-scrollbar */
+/* stylelint-disable-next-line selector-id-pattern */
+#save_status_log::-webkit-scrollbar {
+    height: 4px;
+    width: 4px;
+    border-radius: 2px;
+}
+
+/* stylelint-disable-next-line selector-id-pattern */
+#save_status_log::-webkit-scrollbar-track {
+    background: var(--standard-light-gray);
+    border-radius: 2px;
+}
+
+/* stylelint-disable-next-line selector-id-pattern */
+#save_status_log::-webkit-scrollbar-thumb {
+    background: var(--standard-medium-gray);
+    border-radius: 2px;
+}
+
+/* stylelint-disable-next-line selector-id-pattern */
+#save_status_log::-webkit-scrollbar-thumb:hover {
+    background: #555555;
```

Please update this to use colors from `colors.css` for consistency.
```suggestion
    background: var(--subtle-grey-background);
```